components/calendar/index.js

```diff
@@ -7,7 +7,9 @@ import { API_EVENTS_URL } from '../../common/constants';
 require('es6-promise').polyfill();
 require('isomorphic-fetch');
 
-const TODAY = moment()
+moment.locale('nb');
```

Comment:
I meant index.js in the root folder. 😛 
I usually use the main file to configure things like moment. This way it's easy to find later on instead of being buried in a "random" file.